crates/ethereum/rpc/src/api_builder.rs

```diff
@@ -0,0 +1,48 @@
+use reth_evm::ConfigureEvm;
+use reth_network_api::NetworkInfo;
+use reth_provider::{CanonStateSubscriptions, FullRpcProvider};
+use reth_rpc::EthApi;
+use reth_rpc_builder::{
+    EthApiBuilder, EthApiBuilderCtx, FeeHistoryCacheBuilder, GasPriceOracleBuilder,
+};
+use reth_tasks::{pool::BlockingTaskPool, TaskSpawner};
+use reth_transaction_pool::TransactionPool;
+
+/// Ethereum layer one `eth` RPC server builder.
+#[derive(Default, Debug, Clone, Copy)]
+pub struct EthApiBuild;
```

Comment:
we could name this EthEthApiBuilder :D but that would be silly -.-

Reply:
haha yeah that's why I like to name traits with imperative verbs like `BuildEthApi`, so the noun I reserved for the type